QUARKUS-3249.md

```diff
@@ -0,0 +1,32 @@
+# QUARKUS-3249 Configurable init container
+
+This document is a high level test plan for options to configure init containers in Quarkus 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-3249
+
+## Scope of the testing
+* Check, that flyway task can be disabled
+* Check, that we can use custom image for the run and change its pull strategy
+  * I suppose, we should build the image ourselves, so a Docker file should be provided 
```

Comment:
I can't actually come with a scenario for this from the top of my head, but I think it would be useful to describe the test scenario for this in the test plan itself